wick/expression.py

```diff
@@ -495,19 +495,43 @@ def _print_str(self, with_scalar=True):
             out += tt._print_str(imap)
         return out
 
-    def _einsum_str(self):
+    def _einsum_str(self, exprs_with_space = None, names_with_space = None, space_idx_formatter = None, einsum_str_formatter = None):
```

Comment:
Line is too long, there shouldn't be spaces around the `=` signs in the function definition.